acmw-db-app/components/FormComponents/CompanySearchInput.jsx

```diff
@@ -33,14 +31,22 @@ const CompanySearchInput = (props) => {
             getCompanyInfo(event.target.value).then((data) => {
                 const tempList = []
                 for (var i in data) {
-                    tempList.push(data[i]["cname"])
+                    tempList.push({
+                        label: data[i]["cname"],
+                        value: data[i]["id"]})
                 }
+
                 if(subscribed.current) setCompanyList(tempList)
```

Comment:
Since there are now multiple updates that could potentially be on an unmounted component, I think this should be changed to 2 lines:  `if(!subscribed.current) return` then `setCompanyList(tempList)` to completely quit everything if the user has left the page